src/plugins/network.ts

```diff
@@ -194,6 +226,9 @@ export class NetworkManager {
     }
     if (encodedDataLength >= 0) {
       record.transferSize = encodedDataLength;
```

Comment:
Do we need both values? I'm wondering if `response.body` is sufficient

Reply:
We use `transferSize` in the UI. the `response.body` is just for the ECS spec.